oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/client/TestRegisteredClients.java

```diff
@@ -57,4 +57,16 @@
 				.scope("scope1")
 				.clientSettings(clientSettings -> clientSettings.requireProofKey(true));
 	}
+
+	public static RegisteredClient.Builder registeredClientWithLoopbackRedirection() {
```

Comment:
Please revert and reuse `registeredClient()` and change attributes via returned `Builder`